bfcfg

```diff
@@ -3424,6 +3441,7 @@ while [ "$1" != -- ]; do
     --skip-mfg) skip_mfg=1 ;;
     --cfg2bin|-c) has_cfg2bin=1; infile=$2 ;;
     --bin2cfg|-b) has_bin2cfg=1; infile=$2 ;;
+    --cfg|-f) cfg_file=$1 ;;
```

Comment:
Q: do we need to bump up the `bfcfg` command version?

Reply:
done